src/libraries/Microsoft.Extensions.DependencyModel/src/Microsoft.Extensions.DependencyModel.csproj

```diff
@@ -30,4 +30,8 @@
     <ProjectReference Include="$(LibrariesProjectRoot)System.Text.Json\src\System.Text.Json.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'">
```

Comment:
Even though this will be reverted, I don't think it is appropriate to check for TargetFrameworks that a .csproj doesn't build for. This project builds for `netstandard2.0;net461`. Thus this condition is always `true`.